command/server/server.go

```diff
@@ -174,6 +188,47 @@ func (s *Server) setupMetrics(config *TelemetryConfig) error {
 
 	}
 
+	if config.OpenCollectorEndpoint != "" {
+		// setup open collector tracer
+		ctx := context.Background()
+
+		res, err := resource.New(ctx,
+			resource.WithAttributes(
+				// the service name used to display traces in backends
+				semconv.ServiceNameKey.String("bor"), // TODO: use the service name from the config
+			),
+		)
+		if err != nil {
+			return fmt.Errorf("failed to create open telemetry resource for service " + "bor")
```

Comment:
Could you write this instead:
```
return fmt.Errorf("failed to create open telemetry resource for service: %v", err)
```
We do not need to print the service name since the user should know already to which instance these logs belong, but, we do need to print the specific error.